crates/moonbuild-rupes-recta/src/build_lower/compiler/link_core.rs

```diff
@@ -159,6 +159,8 @@ impl<'a> MooncLinkCore<'a> {
         // WebAssembly exports
         if let Some(exports) = self.exports
             && !exports.is_empty()
+            && !self.test_mode
+        // when it's test mode, we only export test driver utils
```

Comment:
Might worth reviewing how "dumb" the commandline builders should be some time in the future. It's becoming increasingly "smart" as we're fixing bugs. 

I'm not against this, but a rule-of-thumb of "which goes to the wrappers, which goes to the args" would benefit future edits?